CONTRIBUTING.md

```diff
@@ -10,6 +10,7 @@
 * Code the algorithm/DS following the styleguide defined below.
 * Send a PR. 
 * Be sure to not include any compiled binaries in the patch.
+* While sending a PR make sure you follow one PR per issue rule.
```

Comment:
One issue one PR and a single commit :+1:  What u think ? @aviaryan  @singhpratyush @Monal5031